src/themes/_utility.css

```diff
@@ -6,7 +6,10 @@
 
 @supports (scrollbar-gutter: stable) {
   .sl-scroll-lock {
-    scrollbar-gutter: stable !important;
+    scrollbar-gutter: var(--sl-scroll-lock-gutter) !important;
+  }
+
+  .sl-scroll-lock body {
```

Comment:
Should probably add a comment to the code, but by moving the `overflow` to the body, it prevents the "scroll to top" issue mentioned in #1951